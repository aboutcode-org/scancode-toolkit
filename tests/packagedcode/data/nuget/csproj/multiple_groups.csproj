<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
<Project Sdk="Microsoft.NET.Sdk">
    <ItemGroup>
        <Manifest Include="app.manifest" />
        <Content Include="7z.dll">
            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
        </Content>
    </ItemGroup>

    <ItemGroup>
        <PackageReference Include="ByteSize" Version="2.1.1" />
        <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
    </ItemGroup>
    <ItemGroup>
        <ProjectReference Include="..\Files.App.BackgroundTasks\Files.App.BackgroundTasks.csproj" />
    </ItemGroup>

    <ItemGroup>
        <TrimmerRootAssembly Include="Files.App.BackgroundTasks" />
        <TrimmerRootAssembly Include="CommunityToolkit.WinUI.UI.Controls.Primitives" />
    </ItemGroup>

</Project>

