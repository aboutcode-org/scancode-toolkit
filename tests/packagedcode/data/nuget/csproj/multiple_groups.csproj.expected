[
  {
    "type": "nuget",
    "namespace": null,
    "name": "ByteSize",
    "version": "2.1.1",
    "qualifiers": {},
    "subpath": null,
    "primary_language": null,
    "description": null,
    "release_date": null,
    "parties": [],
    "keywords": [],
    "homepage_url": null,
    "download_url": null,
    "size": null,
    "sha1": null,
    "md5": null,
    "sha256": null,
    "sha512": null,
    "bug_tracking_url": null,
    "code_view_url": null,
    "vcs_url": null,
    "copyright": null,
    "holder": null,
    "declared_license_expression": null,
    "declared_license_expression_spdx": null,
    "license_detections": [],
    "other_license_expression": null,
    "other_license_expression_spdx": null,
    "other_license_detections": [],
    "extracted_license_statement": null,
    "notice_text": null,
    "source_packages": [],
    "file_references": [],
    "extra_data": {},
    "dependencies": [],
    "repository_homepage_url": "https://www.nuget.org/packages/ByteSize/2.1.1",
    "repository_download_url": "https://www.nuget.org/api/v2/package/ByteSize/2.1.1",
    "api_data_url": "https://api.nuget.org/v3/registration3/bytesize/2.1.1.json",
    "datasource_id": "nuget_csproj",
    "purl": "pkg:nuget/ByteSize@2.1.1"
  },
  {
    "type": "nuget",
    "namespace": null,
    "name": "CommunityToolkit.Mvvm",
    "version": "8.2.2",
    "qualifiers": {},
    "subpath": null,
    "primary_language": null,
    "description": null,
    "release_date": null,
    "parties": [],
    "keywords": [],
    "homepage_url": null,
    "download_url": null,
    "size": null,
    "sha1": null,
    "md5": null,
    "sha256": null,
    "sha512": null,
    "bug_tracking_url": null,
    "code_view_url": null,
    "vcs_url": null,
    "copyright": null,
    "holder": null,
    "declared_license_expression": null,
    "declared_license_expression_spdx": null,
    "license_detections": [],
    "other_license_expression": null,
    "other_license_expression_spdx": null,
    "other_license_detections": [],
    "extracted_license_statement": null,
    "notice_text": null,
    "source_packages": [],
    "file_references": [],
    "extra_data": {},
    "dependencies": [],
    "repository_homepage_url": "https://www.nuget.org/packages/CommunityToolkit.Mvvm/8.2.2",
    "repository_download_url": "https://www.nuget.org/api/v2/package/CommunityToolkit.Mvvm/8.2.2",
    "api_data_url": "https://api.nuget.org/v3/registration3/communitytoolkit.mvvm/8.2.2.json",
    "datasource_id": "nuget_csproj",
    "purl": "pkg:nuget/CommunityToolkit.Mvvm@8.2.2"
  }
  ]
